server/db/db.js

```diff
@@ -4,7 +4,7 @@ const pkg = require('../../package.json')
 const databaseName = pkg.name + (process.env.NODE_ENV === 'test' ? '-test' : '')
 
 const db = new Sequelize(
-  process.env.DATABASE_URL || `postgres://localhost:5432/${databaseName}`,
+  process.env.DATABASE_URL || `postgres://localhost:5432/${candycrushers}`,
```

Comment:
This is evaluating a variable called candycrushers in the ${}, which I believe will return undefined. Since we changed the name in the package.json file to 'candy-crushers', we can leave the code as is and dynamically pull the database name from that file (it will connect to a db named candy-crushers).